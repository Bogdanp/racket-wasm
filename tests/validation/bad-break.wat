(module
 (type (func (result i32 i32)))
 (export "main" (func 0))
 (func
  (result i32)
  (i32.const 1)
  (block
   (block
    (block
     (result i32)
     (br 0))))
  (i32.add)))