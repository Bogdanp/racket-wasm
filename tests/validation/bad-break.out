(#f . "arity error\n  expected: [i32]\n  found: []\n  at: (instr:br 0)\n  within: (instr:block (functype '() '(i32)) (vector (instr:br 0)))\n  within: (instr:block (functype '() '()) (vector (instr:block (functype '() '(i32)) (vector (instr:br 0)))))\n  within: (instr:block (functype '() '()) (vector (instr:block (functype '() '()) (vector (instr:block (functype '() '(i32)) (vector (instr:br 0)))))))\n  within: (funcidx 0)")